ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -192,11 +247,40 @@
                 "\n        foreach (var variable in computedNotExpected)",
                 "\n            errors.Add(new BenchmarkTestResult(\"Extra computed variable for: Partition \" + variable.Partition + \", \" + variable.ToIdentityString()));",
                 "\n",
-                "\n    var comparer = EqualityComparer<IfrsVariable>.Instance; ",
+                "\n    var comparer = IfrsVariableComparer.Instance(true); ",
                 "\n    foreach (var benchmarkVariable in expected) {",
-                "\n        var computedVariable = computed.FirstOrDefault(y => comparer.Equals(benchmarkVariable,y))?? null;",
-                "\n        if(computedVariable == null || Math.Abs(benchmarkVariable.Value + computedVariable.Value) < BenchmarkPrecision) continue;",
-                "\n        errors.Add(new BenchmarkTestResult(\"Value does not match for Variable: Partition \" + benchmarkVariable.Partition + \", \" + benchmarkVariable.ToIdentityString(), -benchmarkVariable.Value, computedVariable.Value));",
+                "\n        var computedVariable = computed.FirstOrDefault(y => comparer.Equals(benchmarkVariable,y));",
+                "\n        if(computedVariable == default(IfrsVariable)) continue;",
+                "\n        ",
+                "\n        if(Math.Abs(benchmarkVariable.Value + computedVariable.Value) > BenchmarkPrecision){",
+                "\n            errors.Add(new BenchmarkTestResult(\"Value does not match for Variable: Partition \" + benchmarkVariable.Partition + \",\" + benchmarkVariable.ToIdentityString(),",
+                "\n                                 -benchmarkVariable.Value, ",
+                "\n                                 computedVariable.Value));",
+                "\n            }",
+                "\n",
+                "\n    if(isProjectionTest)",
```

Comment:
imo this should not be needed. Controlling properly the number of projection computed and those present in the BM, we should always compare every ValueN

Reply:
Removed isProjectionTest boolean. Organized code according to suggestions.